test/features/deploy_kogito_build_vs_code.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Deploy Kogito Build - after assets processed by VS code
+  
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+    And Project kie-asset-library is cloned
+    And Project kie-asset-library is built by maven with configuration:
+      | profile | dmn                                    |
+      | option  | -Dactive.structures=quarkus            |
+      | option  | -Dactive.config.sets=scesim            |
+      | native  | disabled                               |
+    Then Project "<project>" is generated in temporary folder
```

Comment:
The step text looks misleading. On the first looks I would guess that it reference the kie-asset-library project.
Maybe it could be changed to something like `And Project "<project>" is generated kie-asset-library`